src/test/java/com/training/services/tests/GAAccountTest.java

```diff
@@ -93,14 +94,17 @@ public void testAuthenticateWrongUsername() {
      */
     @Test
     public void testGuestAccountValidation() {
-        Map<String, Object> headerMap = new HashMap();
         headerMap.put(map.get("AppKeyHeader"), map.get("AppKeyValue"));
         headerMap.put(map.get("ContentTypeHeader"), map.get("ContentTypeValue"));
 
+        RequestBody requestBody = new RequestBody();
+        requestBody.setEmailKey("email");
+        requestBody.setEmailValue("testPranav@api.com");
+
         com.training.services.ga.validate.Response gaValidationResponse = new RestEngine()
                 .getResponsePost(map.get("base_url") + "/validation"
                         , headerMap
-                        , "{\"email\": \"testPranav@api.com\"}")
+                        , "{\"" + requestBody.getEmailKey() + "\": \"" + requestBody.getEmailValue() + "\"}")
```

Comment:
instead of passing it as individual strings, apply the below logic : 
Pojo class -> serialise into Json -> toString() and then pass it to RestEngine